addons/payment_stripe/controllers/main.py

```diff
@@ -171,3 +169,40 @@ def _verify_webhook_signature(self, webhook_secret):
             return False
 
         return True
+
+    # -------------------------------------------
+    # Stripe Connect Onboarding
+    # -------------------------------------------
+    @http.route('/payment/stripe/onboarding/return/<int:acquirer_id>', type='http', auth='public', csrf=False)
```

Comment:
In case the line is still > 100 chars after extracting the route, please wrap it like this:
```suggestion
    @http.route(
        '/payment/stripe/onboarding_return/<int:acquirer_id>', type='http', auth='public',
        csrf=False
    )
```